electronics/README.asciidoc

```diff
@@ -0,0 +1,69 @@
+# GibiBot Electronics Documentation
+
+## Information
+
+The mainboard is controlled through a STM32G441KBT6 microcontroller. It uses the USB connector to communicate with computer. The microcontroller is powered by the USB connector through a 5V to 3.3V voltage regulator(NCP164CSN330T1G). It can be programmed through the programmer connector with a STLINK programmer.
+
+The board has two isolated sides which have both have their own seperate ground planes. 2 MAX14930DASE+ isolators and 1 MAX12930EASA+ isolator connect both sides. The microcontroller sends signals through the isolators to the motor drivers, servos and the thrower.
+
+The microcontroller side has a debug LED which can be controlled through the microcontroller.
+
+The driver side has 3 PDRV8243HRXYQ1 motor drivers which can power 3 motors. The motor drivers need an nsleep signal from the microcontroller to wake up. The drivers are controllerd with a direction signal and a PWM signal. The direction pin determines which way the motor spins.
+
+The driver side also has a switching regulator (TPS62933DRLR) which converts the input voltage to 5V for the isolators and servos.
+
+### Microcontroller side:
+* Powered trough a micro USB connector.
+* It has USB protection.
+
+### The motor driver side:
+* Input voltage up to 16.8V.
+* Input has reverse voltage protection.
+* Motor driver current output up to 2.77A.
+
+## PCB
+image:https://user-images.githubusercontent.com/112662827/212202425-6cf1347a-7b41-4bcb-865b-d04923f3c480.png[]
+
+## Connector Pinouts
+
+### J6 - Programmer Connector
+1. 3.3V
+2. SWDIO
+3. GND
+4. SWCLK
+5. NC
+6. NC
+7. GND
+8. NC
+9. GND
+10. NRST

```

Comment:
I have missed it during design review, but pin 5 should be GND, not pin 7,

Reply:
Fixed.